### ISSUE TITLE
FileGlobs: Extract ignore from .npmignore file

### ISSUE THREAD

**Issue by satwikkansal** [coala/coala-quickstart#109]
Some developers use `.npmignore` file to keep stuff out of their npm package. If there's no `.npmignore` file, but there is a `.gitignore` file, then npm will ignore the stuff matched by the `.gitignore` file.

 Like git, npm looks for `.npmignore` and `.gitignore` files in all subdirectories of your package, not only the root directory. (There's an issue https://github.com/coala/coala-quickstart/issues/108 for this functionality)

`.npmignore` files follow the same pattern rules as `.gitignore` files.

difficulty/low

**Comment by prachi1210** [comment]
Hi, I'd like to work on this issue.

**Comment by satwikkansal** [comment]
Hey, use cobot to assign yourself :)

**Comment by prachi1210** [comment]
@satwikkansal just to clarify, I need to recursively look for .npmignore files in all subdirs and extract globs, right?

**Comment by satwikkansal** [comment]
@prachi1210 yeah right. There's another PR for git ignore, you can take help from that ;)

**Comment by satwikkansal** [comment]
Link to the PR: https://github.com/coala/coala-quickstart/pull/110

**Comment by djmgit** [comment]
@prachi1210 hi, are you still working on this? If not I would like to take this up.
